src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayExtensions.cs

```diff
@@ -54,6 +54,46 @@ public static string ToDisplayString(this ImmutableArray<SymbolDisplayPart> part
             }
         }
 
+        /// <summary>
+        /// Converts an ArrayBuilder of <see cref="SymbolDisplayPart"/>s to a string.
+        /// </summary>
+        /// <param name="parts">The array of parts.</param>
+        /// <returns>The concatenation of the parts into a single string.</returns>
+        [PerformanceSensitive("https://github.com/dotnet/roslyn/pull/67203", AllowImplicitBoxing = false)]
+        internal static string ToDisplayString(this ArrayBuilder<SymbolDisplayPart> parts)
+        {
+            if (parts is null)
+            {
+                throw new ArgumentException("parts");
```

Comment:
```suggestion
                throw new ArgumentNullException(nameof(parts));
```

Reply:
Good catch, changed here and the code it was copied from